x-pack/plugin/sql/sql-action/src/main/java/org/elasticsearch/xpack/sql/action/SqlSemVersion.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.sql.action;
+
+import org.elasticsearch.core.Nullable;
+import org.elasticsearch.xpack.sql.proto.SqlVersion;
+
+/**
+ * This is a specialization of the SqlVersionId that maps to the SemVer versioning scheme in use by the SQL API (wrapping SqlVersion).
+ */
+public final class SqlSemVersion extends SqlVersionId {
```

Comment:
How about `SqlReleaseVersion` instead? `Sem` is a bit jargon-y

Reply:
I've dropped it entirely.